gui/public/i18n/en/translation.ftl

```diff
@@ -1248,14 +1249,14 @@ onboarding-scaled_proportions-done-title = Body proportions set
 onboarding-scaled_proportions-done-description = Your body proportions should now be configured based on your height.
 
 ## Stay Aligned setup
-onboarding-stay_aligned-title = Stay Aligned
-onboarding-stay_aligned-description = Configure Stay Aligned to keep your trackers aligned.
+onboarding-stay_aligned-title = { -stay_aligned }
+onboarding-stay_aligned-description = Configure { -stay_aligned } to keep your trackers aligned.
 onboarding-stay_aligned-put_trackers_on-title = Put on your trackers
 onboarding-stay_aligned-put_trackers_on-description = To save your resting poses, we'll use the trackers you just assigned. Put on all your trackers, you can see which are which in the figure to the right.
-onboarding-stay_aligned-put_trackers_on-trackers_warning = You have fewer than 5 trackers currently connected and assigned! This is the minimum amount of trackers required for Stay Aligned to function properly.
+onboarding-stay_aligned-put_trackers_on-trackers_warning = You have fewer than 5 trackers currently connected and assigned! This is the minimum amount of trackers required for { -stay_aligned } to function properly.
 onboarding-stay_aligned-put_trackers_on-next = I have all my trackers on
 onboarding-stay_aligned-verify_mounting-title = Check your Mounting
-onboarding-stay_aligned-verify_mounting-step-0 = Stay Aligned requires good mounting. Otherwise, you won't get a good experience with Stay Aligned.
+onboarding-stay_aligned-verify_mounting-step-0 = { -stay_aligned } requires good mounting. Otherwise, you won't get a good experience with { -stay_aligned }.
 onboarding-stay_aligned-verify_mounting-step-1 = 1. Move around while standing.
 onboarding-stay_aligned-verify_mounting-step-2 = 2. Sit down and move your legs and feet.
 onboarding-stay_aligned-verify_mounting-step-3 = 3. If your trackers aren't in the right place, press "Redo Mounting Calibration"
```

Comment:
cam you make "Redo Mounting Calibration" just refer to the string via a variable